robot_toolkit/config/dope_config_pose.yaml

```diff
@@ -0,0 +1,176 @@
+topic_camera: "/dope/webcam/image_raw"
```

Comment:
in case, we have not provided the reference for this file, please ensure refer the place we got this file from.